content/v2.0/reference/api/client-libraries/js.md

```diff
@@ -1,12 +1,187 @@
 ---
-title: InfluxDB JavaScript client library
+title: JavaScript client library
+seotitle: InfluxDB JavaScript client library
 list_title: JavaScript
-description: Use the JavaScript client library to interact with InfluxDB.
-external_url: https://github.com/influxdata/influxdb-client-js
+description: >
+  Use the JavaScript client library to interact with InfluxDB.
 menu:
   v2_0_ref:
     name: JavaScript
     parent: Client libraries
-    url: https://github.com/influxdata/influxdb-client-js
+v2.0/tags: [client libraries, JavaScript]
 weight: 201
 ---
+
+Use the [InfluxDB JavaScript client library](https://github.com/influxdata/influxdb-client-js) to integrate InfluxDB into JavaScript scripts and applications. This client supports both client-side (browser) and server-side (NodeJS) environments. 
+
+This guide presumes some familiarity with JavaScript, browser environments, and InfluxDB.
+If just getting started, see [Get started with InfluxDB](/v2.0/get-started/).
+
+## Before you begin
+
+1. Install Go 1.3 or later](https://golang.org/doc/install)
```

Comment:
I think this got pasted into the wrong guide. This should be on the Go guide :)

Reply:
oof thank you.